strawberryfields/device.py

```diff
@@ -53,12 +57,12 @@ def target(self) -> str:
 
     @property
     def layout(self) -> Optional[str]:
-        """str: Returns a string containing the Blackbird circuit layout."""
+        """str: A string containing the Blackbird circuit layout."""
```

Comment:
As per the [PennyLane development guide](https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html), should we preserve the "Returns a ..." wording?

Reply:
It was changed to follow the rest of the docstrings in the file. Does the guide say that and action "Returns a ..." is preferred to a descriptive first line? I could only find that is should be a "Single sentence that summarizes the function.". 🤔

Reply:
Oh, sorry, I was specifically referring to:
> Document all methods and properties with docstrings below the method declaration, as you would for functions above.

Reply:
Oh, I see. Yes, definitely. That makes sense. I guess I was just a bit lazy here. 😅

Reply:
@Mandrenkov which part of the documentation guide are you referring to?

Reply:
@josh146, I'm referring to the _Documentation_ page of the PennyLane development guide under the [**Classes** section](https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#classes) of the **Docstrings** header.

Reply:
Okay, it turns out I misunderstood the intent of that line in the PennyLane development guide. Carry on! 🙂